compio-driver/src/iocp/op.rs

```diff
@@ -781,12 +775,12 @@ static WSA_RECVMSG: OnceLock<LPFN_WSARECVMSG> = OnceLock::new();
 
 /// Receive data and source address with ancillary data into vectored buffer.
 pub struct RecvMsg<T: IoVectoredBufMut, C: IoBufMut, S> {
+    msg: WSAMSG,
     addr: SOCKADDR_STORAGE,
-    addr_len: socklen_t,
     fd: SharedFd<S>,
     buffer: T,
     control: C,
-    control_len: u32,
+    slices: Vec<IoSliceMut>,
```

Comment:
I don't think `slices` is needed.

Reply:
I don't actually understand how IOCP works :(

Reply:
I'm not that sure actually, but I think it will be better to try without it first.

Reply:
You are right. Seems that they do not care lpBuffers.